src/features/public/Donations/screens/ProjectDetails.tsx

```diff
@@ -18,23 +18,26 @@ interface Props {
 }
 
 function ProjectDetails({ project }: Props): ReactElement {
+
+
   const [rating, setRating] = React.useState<number | null>(2);
 
   const progressPercentage =
     (project.countPlanted / project.countTarget) * 100 + '%';
   const ImageSource = project.image
     ? getImageUrl('project', 'large', project.image)
```

Comment:
– image medium should suffice for general purpose.

large can be loaded for retina image. Not sure if next has built in feature, but css norm is: to supply srcset.
eg:
`<img src="img/projectpic@1x.png" srcset="img/projectpic@2x.png 2x, img/projectpic@3x.png 3x" />`